hi-ml-histopathology/src/histopathology/configs/classification/BaseMIL.py

```diff
@@ -65,58 +64,35 @@ class BaseMIL(LightningContainer):
     batch_size: int = param.Integer(16, bounds=(1, None), doc="Number of slides to load per batch.")
     encoding_chunk_size: int = param.Integer(0, doc="If > 0 performs encoding in chunks, by loading"
                                                     "enconding_chunk_size tiles per chunk")
-    is_caching: bool = param.Boolean(False, doc="If True, cache the encoded tile features "
-                                     "(disables random subsampling of tiles). "
-                                     "If False (default), load the tiles without caching "
-                                     "(enables random subsampling of tiles).")
     # local_dataset (used as data module root_path) is declared in DatasetParams superclass
 
     def __init__(self, **kwargs: Any) -> None:
         super().__init__(**kwargs)
         self.best_checkpoint_filename = "checkpoint_max_val_auroc"
         self.best_checkpoint_filename_with_suffix = self.best_checkpoint_filename + ".ckpt"
-        self.checkpoint_folder_path = "outputs/checkpoints/"
-        self.callbacks = self.get_monitoring_callbacks(monitor="val/auroc", mode="max")
+        self.checkpoint_folder_path = f"{self.outputs_folder}/checkpoints/"
```

Comment:
I believe this attribute is now used with different purposes. In `download_ssl_checkpoint()`, this needs to be as before (`"outputs/checkpoints/"`), because it's actually the _remote_ checkpoint location on AML, not a local path. And see my other suggestion for the checkpoint callback (using the `LightningContainer.checkpoint_folder` property instead). I suggest we get rid of `self.checkpoint_folder_path`.